articles/ansible/cache-for-redis-configure.md

```diff
@@ -107,7 +107,7 @@ It can take several minutes to scale a cache. The following code tells Ansible w
 
 Similar to the task to provision Azure Cache for Redis, output like the following message is normal:
 
-```Ouput
+```Output
```

Comment:
```suggestion
```output
```

Reply:
@KarlErickson note that this inconsistency is in many files Output vs output.

Reply:
The lowercase string "output" in on our list of recognized values, which is why I changed that. If a value isn't recognized, then it's used as-is. So "Output" will work, but it might not have the exact same effect as "output". (For other strings, such as "azurecli", one of the effects is code colorization, but I doubt there is any kind of special colorization that would be standard for something as generic as "output"!)